src/index.d.ts

```diff
@@ -0,0 +1,25 @@
+export interface WebhookEvent {
+  id: string | string[] | undefined;
+  name: string | string[] | undefined;
+  payload: Payload;
+  signature: string | string[] | undefined;
+}
+
+type Payload = {
+  action: string;
+};
+
+export interface EventHandlerOptions {
+  path?: string;
+  secret?: string;
+  transform?: (value: WebhookEvent) => WebhookEvent | PromiseLike<WebhookEvent>;
+}
+
+type Hooks = {
+  [key: string]: Function[];
+};
+
+export interface EventState extends EventHandlerOptions {
+  eventHandler?: any;
+  hooks: Hooks;
+}
```

Comment:
In the other Octokit repos I have a `src/types.ts` file if I need to define any internal types. If need to export the types, I just export them in `src/index.ts`. I don't have any `*.d.ts` files in the other Octokit repos. I wonder if you could follow that convention here?

Reply:
> In the other Octokit repos I have a `src/types.ts` file if I need to define any internal types. If need to export the types, I just export them in `src/index.ts`. I don't have any `*.d.ts` files in the other Octokit repos. I wonder if you could follow that convention here?

Of course! Sorry to not check the file structure for types in other repos 🙏 . Applying it!